spec/models/embedded_ansible_worker/runner_spec.rb

```diff
@@ -13,26 +13,35 @@
       described_class.new(:guid => worker_guid)
     }
 
-    context "#update_embedded_ansible_manager" do
+    context "#update_embedded_ansible_provider" do
+      before do
+        EvmSpecHelper.local_guid_miq_server_zone
+        MiqDatabase.seed
+        MiqDatabase.first.ansible_admin_password = "secret"
```

Comment:
Yeah, these are all in the authentications table. They were added [here](https://github.com/ManageIQ/manageiq/pull/13733/files#diff-fb990620e53ea4cfdee9defa0ff8882eL52) and are getting a bit of a clean up [here](https://github.com/ManageIQ/manageiq/pull/13749).